doc/index.rst

```diff
@@ -70,13 +70,23 @@ Now you can build and run the docker container:
 
 First Steps
 -----------
-    
+
+Set Alias to Run Mailadm CLI Commands Easier (optional)
```

Comment:
it's not optional if the rest of the docs depend on the alias. Just strike the "optional".